### DISCUSSION TITLE
Search a product by array using graphQL?

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#560] [Help] [answered]
Welcome,

Is there an option to filter products by name i.e `['product 1', 'product 2', 'product 3']`?

Currently the [docs](https://developers.swell.is/frontend-api/products#search-for-products) shows only search for one string is supported?

**Comment by ericingram** [comment, marked as answer]
There are a couple ways to do it depending on what you’re trying to achieve. You can use $or with multiple $regex operators in a single query (note the search parameter used regex also), or use an include or batch to make multiple search queries at a time. Not recommended with a large number of products though.

**Reply by tarikrital** [reply]
Giving an example makes the answer very clear.

Thank you, @ericingram
